sensor/tag_based_pnp_calibrator/README.md

```diff
@@ -0,0 +1,118 @@
+| `use_rectified_image`                         | `bool`                | `true`                           | Flag to determine whether rectified images should be used in the calibration process.                                                                     |
+| `calibration_crossvalidation_training_ratio`  | `double`              | `0.7`                            | The ratio of data used for training versus testing during the calibration's cross-validation process.                                                     |
+| `calibration_convergence_min_pairs`           | `int`                 | `9`                              | The minimum number of AprilTag and LidarTag detection pairs required to consider the calibration process as potentially converged.                        |
+| `calibration_convergence_min_area_percentage` | `double`              | `0.005`                          | Minimum percentage of the area that needs to be covered by detection.                                                                                     |
+| `min_pnp_points`                              | `int`                 | `8`                              | Minimum number of points required for the Perspective-n-Point problem used in calibration to solve the pose estimation.                                   |
+| `min_convergence_time`                        | `double`              | `6.0`                            | Minimum time required for the calibration process to be considered as converged.                                                                          |
+| `max_no_observation_time`                     | `double`              | `3.0`                            | If the time between new detection and the last observed detection are larger than `max_no_observation_time`, remove the detection from active hypotheses. |
+| `new_hypothesis_distance`                     | `double`              | `1.5`                            | Distance threshold for creating a new hypothesis.                                                                                                         |
+| `tag_ids`                                     | `std::vector<int>`    | `[0, 1, 2, 3, 4, 5]`             | List of tag IDs that are used in the calibration process.                                                                                                 |
+| `tag_sizes`                                   | `std::vector<double>` | `[0.6, 0.6, 0.6, 0.6, 0.6, 0.6]` | Payload sizes of the tags corresponding to the IDs in meters.                                                                                             |
+| `lidartag_max_convergence_translation`        | `double`              | `0.05`                           | Maximum translation in meters allowed for a lidar tag detection hypothesis to be considered as converged.                                                 |
```

Comment:
The "convergence" parameters are not well explained.
How do you use the parameter? With exactly what you compare it to determine convergence?

Reply:
I kind of want to write like this "`lidartag_max_convergence_translation` serves as a threshold to define the maximum allowable standard deviation of position estimates (in meters) for the hypothesis being tracked. If the estimated position's uncertainty (standard deviation) is below this threshold, the hypothesis is considered stable enough to be "converged." but I am still not sure if it is the description you want. 

Also, regarding to the code in lidartag_hypothesis.cpp line 205.
The max_translation_dot_cov is always 0.0, I am not sure if this is correct or not.

`  if (
    std::sqrt(max_translation_cov) > convergence_translation_ ||
    std::sqrt(max_translation_dot_cov) > convergence_translation_dot_ ||
    std::sqrt(max_rotation_cov) > convergence_rotation_ ||
    std::sqrt(max_translation_dot_cov) > convergence_rotation_dot_ ||
    getSpeed() > convergence_translation_dot_) {
    return false;
  }`

If possible, could you write these parameters descriptions as I think it's a bit hard for me to explain them perfectly. 
Thanks!

Reply:
Addressed in [ca0f677](https://github.com/tier4/CalibrationTools/pull/164/commits/ca0f677e403501ca7f66c47f0ba0c0eb42bfbdf1)